tests/test_job_translator.py

```diff
@@ -134,6 +134,15 @@ def test_single_docket_json_to_job():
     assert test_job == docket_job
 
 
+def test_none_json_to_job():
```

Comment:
Again, if you're going to remove what I mentioned in my other comment, this test of scenario is not going to happen